components/script/dom/storage.rs

```diff
@@ -196,7 +196,7 @@ impl Runnable for StorageEventRunnable {
         let root_context = script_thread.root_browsing_context();
         for it_context in root_context.iter() {
             let it_window = it_context.active_window();
-            assert!(UrlHelper::SameOrigin(&ev_url, &it_window.get_url()));
+            //assert!(UrlHelper::SameOrigin(&ev_url, &it_window.get_url()));
```

Comment:
Argh, no. I'll remove it. I keep having to make this change locally because this assertion fires all the time when saving pages locally to disk.